### ISSUE TITLE
Takes long time to close large xml files, even when not saving

### ISSUE THREAD

**Issue by doorman02** [DotJoshJohnson/vscode-xml#143]
Issue Type
Bug

Description
I have a 5 mb xml file that I need to view. When I close the file it takes couple of seconds to close.
This happends only after I format the xml file using the shortcut alt + shift + f 

VS Code Info
VS Code version: Code 1.20.1 (f88bbf9, 2018-02-13T15:34:36.336Z)
OS version: Windows_NT x64 10.0.16299

System Info
Extensions (2)
Reproduces only with extensions

**Comment by DotJoshJohnson** [comment]
I was able to reproduce this without the XML Tools extension using the steps below:

1. Open a large (8MB) XML file in VS Code.
2. Copy the XML to an external formatter (I used Notepad++'s XML Tools extension).
3. Paste the result into VS Code, replacing the original XML.
4. Close the large XML file in VS Code.

Note that the size of the XML data can more than double, depending on the structure due to the whitespace being added during formatting. I was even able to reproduce without using an external formatter by just hitting "Enter" a few dozen times in the middle of the document, then closing it.
